docs/cog_guides/core.rst

```diff
@@ -0,0 +1,3393 @@
+    - ``[p]set colour blurple``
+    - ``[p]set colour 0x5DADE2``
+    - ``[p]set color 0x#FDFEFE``
+    - ``[p]set color #7F8C8D``
+
+**Arguments:**
+
+- ``[colour]`` The colour to use for embeds. Leave blank to set to the default value (red).
+
+.. _core-command-set-competing:
+
+"""""""""""""
+set competing
+"""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set competing [competing]
+
+**Description**
+
+Sets Red's competing status.
+
+This will appear as ``Competing in <competing>``.
+
+Maximum length for a competing status is 128 characters.
+
+Examples:
+    - ``[p]set competing`` - Clears the activity status.
+    - ``[p]set competing London 2012 Olympic Games``
+
+**Arguments:**
+
+- ``[competing]`` The text to follow ``Competing in``. Leave blank to clear the current activity status.
+
+.. _core-command-set-custominfo:
+
+""""""""""""""
+set custominfo
+""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set custominfo [text]
+
+**Description**
+
+Customizes a section of ``[p]info``.
+
+The maximum amount of allowed characters is 1024.
+Supports markdown, links and "mentions".
+
+Link example: ``[My link](https://example.com)``
+
+Examples:
+    - ``[p]set custominfo >>> I can use **markdown** such as quotes, ||spoilers|| and multiple lines``
+    - ``[p]set custominfo Join my [support server](discord.gg/discord)``
+    - ``[p]set custominfo`` - Removes custom info text.
+
+**Arguments:**
+
+- ``[text]`` The custom info text.
+
+.. _core-command-set-deletedelay:
+
+"""""""""""""""
+set deletedelay
+"""""""""""""""
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set deletedelay [time]
+
+**Description**
+
+Set the delay until the bot removes the command message.
+
+Must be between -1 and 60.
+
+Set to -1 to disable this feature.
+
+This is applied only the current server and not globally.
+
+Examples:
+    - ``[p]set deletedelay`` - Shows the current delete delay setting.
+    - ``[p]set deletedelay 60`` - Sets the delete delay to the max of 60 seconds.
+    - ``[p]set deletedelay -1`` - Disables deleting command messages.
+
+**Arguments:**
+
+- ``[time]`` The seconds to wait before deleting the command message. Use -1 to disable.
+
+.. _core-command-set-description:
+
+"""""""""""""""
+set description
+"""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set description [description]
+
+**Description**
+
+Sets the bot's description.
+Use without a description to reset.
+This is shown in a few locations, including the help menu.
+
+The maximum description length is 250 characters to ensure it displays properly.
+
+The default is "Red V3".
+
+Examples:
+    - ``[p]set description`` - Resets the description to the default setting.
+    - ``[p]set description MyBot: A Red V3 Bot``
+
+**Arguments:**
+
+- ``[time]`` The seconds to wait before deleting the command message. Use -1 to disable.
```

Comment:
This shouldn't be here

Reply:
Added the proper argument name and description.